docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -377,7 +377,7 @@ In the comments section, please provide the relevant details: the artifacts, if
 
 ## References
 
-- Deployed contracts addresses https://docs.lido.fi/deployed-contracts/#lido-on-l2
+- Deployed contracts addresses https://docs.lido.fi/deployed-contracts/#lido-multichain
 - LOL (Liquidity Observation Labs) https://research.lido.fi/t/liquidity-observation-lab-lol-liquidity-strategy-and-application-to-curve-steth-eth-pool/5335
 - Lido L2 reference bridging contracts (Arbitrum and Optimism) https://github.com/lidofinance/lido-l2
```

Comment:
Should we rename this as well?

Reply:
I believe we should, why not?

Reply:
Not sure. Multichain includes other L1s, but this project intended exclusively for L2. So, I'd rather not

Reply:
Agh, I misread it, sorry; it should stay the same bc it's a project name for the particular Arbi/Opti bridges ofc :)